lib/const.rb

```diff
@@ -1,11 +1,16 @@
 # Defines common constants used in different parts of crew
 
-CREW_VERSION = '1.0.6'
+CREW_VERSION = '1.1.0'
 
 ARCH = `uname -m`.strip
 ARCH_LIB = if ARCH == 'x86_64' then 'lib64' else 'lib' end
 
-CREW_PREFIX = '/usr/local'
+if ENV['CREW_PREFIX'].to_s == ''
+  CREW_PREFIX = '/usr/local'
+else
+  CREW_PREFIX = ENV['CREW_PREFIX']
+  @pkg.build_from_source = true
+end
```

Comment:
Probably a better way:
```
CREW_PREFIX = '/usr/local'
if ENV['CREW_PREFIX'].to_s != '' then
  CREW_PREFIX = ENV['CREW_PREFIX']
  @pkg.build_from_source = true
end
```

Reply:
Actually, this might not work since you cannot reassign constants in ruby.  Nevermind.